### PR TITLE
fix price field

### LINKED ISSUE #165: Fix price field on Add Activity Screen
The app crashes when adding a dot the price field. Either fix this or open a custom keyboard with numbers only.
Also, inserting a number using the dot format is not very user friendly for now

### PR DESCRIPTION
**Pull request by mirceagavrizi**
## Summary

I fixed the price field of the activity screen. You can input only numbers and with maximum 2 digits. 

## Changes

- **Models:** estimatedPrice field logic in AddActivityScreen has been updated
- **Bug Fixes/Improvements:** #165

## Checklist

Ensure all / most of these are checked before the pull request is submitted.
- [x] This PR resolves #165 .
- [ ] Tests have been added for new functionality.
- [ ] Screenshots or UI changes have been attached (if applicable).
- [x] Documentation has been updated (if applicable).

##  Testing

Explain how this feature was tested and what kind of testing (unit, integration, UI) has been done. Mention platforms or configurations used for testing.
- **Manual Testing:**
    - [x] Tested on emulator / physical device.
- **Unit Tests**
    - [ ] Added units tests for this

If you haven't added tests, please create a new issue and assign it.

##  Related Issues/Tickets

Closes #165

### CODE REVIEW

File: app/src/main/java/com/android/voyageur/ui/trip/AddActivity.kt [hunk 1]
```diff
@@ -360,11 +363,31 @@ fun AddActivityScreen(tripsViewModel: TripsViewModel, navigationActions: Navigat
                   }
                 }
 
+                fun formatPrice(price: Double): String {
+                  val symbols =
+                      DecimalFormatSymbols.getInstance(Locale.getDefault()).apply {
+                        groupingSeparator = '\''
+                      }
+                  val decimalFormat =
```
Comment (ioanajianu) [inline app/src/main/java/com/android/voyageur/ui/trip/AddActivity.kt:371]: Hi, I'm not sure what you're trying to do here. Can you give some examples and how they should be displayed in each case?

Comment (mirceagavrizi) [reply app/src/main/java/com/android/voyageur/ui/trip/AddActivity.kt:371]: Sure, for example 101053 should be displayed as 101'053 and 1503.76 should be displayed as 1'503.76.

Comment (ioanajianu) [reply app/src/main/java/com/android/voyageur/ui/trip/AddActivity.kt:371]: I see, thanks. From what I tested the ' doesn't get displayed.
I don't think that's a very important issue for now, the main thing we wanted was for the app not to crash when certain characters are introduced.
So you can remove this part of the code for now if you cannot make it work

Comment (mirceagavrizi) [reply app/src/main/java/com/android/voyageur/ui/trip/AddActivity.kt:371]: Ok, I guess I can remove this for now and I'll try to solve it in the next sprint.

**Review by ioanajianu — APPROVED** [review]
Thanks for removing what I suggested!
To further improve this, in the future we can try setting a limit to the number of digits of the price, so that we don't use really big numbers within the app.
LGTM!

**Pull request merged**